atom/browser/atom_resource_dispatcher_host_delegate.cc

```diff
@@ -134,22 +134,31 @@ AtomResourceDispatcherHostDelegate::CreateClientCertStore(
 }
 
 bool AtomResourceDispatcherHostDelegate::ShouldInterceptResourceAsStream(
-    net::URLRequest* request,
-    const base::FilePath& plugin_path,
-    const std::string& mime_type,
-    GURL* origin,
-    std::string* payload) {
-  const content::ResourceRequestInfo* info =
-      content::ResourceRequestInfo::ForRequest(request);
-  if (mime_type == "application/pdf" && info->IsMainFrame()) {
-    *origin = GURL(kPdfViewerUIOrigin);
-    content::BrowserThread::PostTask(
-        BrowserThread::UI, FROM_HERE,
-        base::Bind(&OnPdfResourceIntercepted, request->url(),
-                   info->GetWebContentsGetterForRequest()));
-    return true;
-  }
-  return false;
+	net::URLRequest* request,
+	const base::FilePath& plugin_path,
+	const std::string& mime_type,
+	GURL* origin,
+	std::string* payload) {
+	const content::ResourceRequestInfo* info =
+		content::ResourceRequestInfo::ForRequest(request);
+
+	int render_process_host_id;
+	int render_frame_id;
+	info->GetAssociatedRenderFrame(&render_process_host_id, &render_frame_id);
```

Comment:
Better to verify the result of `GetAssociatedRenderFrame` method before proceeding with retrieving the `frame_tree_node_id`.